### PR TITLE
feat: add request to add a feedstock output

### PR DESCRIPTION
**Pull request by beckermr**
This PR adds code to enable adding feedstock outputs. This will make it easier for folks to do this.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge/core @jaimergp this is ready for review!

File: README.md [hunk 1]
```diff
@@ -72,3 +72,9 @@ This workflow allows users to request a copy once the manual review has been pas
 To do so, please create a new `.yml` file in the `requests` folder. Check `examples/example-cfep-3.yml` for the required metadata.
 
 For provenance and transparency, the PR description must include a link to the original PR and the logs, along with the artifact(s) to be reviewed.
+
+## Add a package output to a feedstock
+
+By default, `conda-forge` feedstocks cannot push packages to our channel that another feedstock makes. If you need to move a package
+build from one feedstock to another, you should request the output be added to the new feedstock via this repository. An example request
+is located in [examples/example-add-output.yml](examples/example-add-feedstock-output.yml). Make a PR putting your `.yml` request file in the `requests` directory and the `conda-forge/core` team will review it.
```
Comment (jaimergp) [inline README.md:80]:
```suggestion
is located in [examples/example-add-feedstock-output.yml](examples/example-add-feedstock-output.yml). Make a PR putting your `.yml` request file in the `requests` directory and the `conda-forge/core` team will review it.
```

File: conda_forge_admin_requests/feedstock_outputs.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                contents.sha,
+            )
+            print(f"    output {pkg_name} added for feedstock conda-forge/{feedstock}-feedstock", flush=True)
+        else:
+            print(f"    output {pkg_name} already exists for feedstock conda-forge/{feedstock}-feedstock", flush=True)
+
+
+def check(request):
+    action = request["action"]
+    assert action in ("add_feedstock_output",)
```
Comment (jaimergp) [inline conda_forge_admin_requests/feedstock_outputs.py:46]:
```suggestion
    assert action == "add_feedstock_output"
```

File: conda_forge_admin_requests/feedstock_outputs.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+    assert "feedstock_to_output_mapping" in request
```
Comment (jaimergp) [inline conda_forge_admin_requests/feedstock_outputs.py:48]: I think we should also check it's not empty


```suggestion
    assert request.get("feedstock_to_output_mapping") 
```

File: conda_forge_admin_requests/feedstock_outputs.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+            r = requests.head(
+                f"https://github.com/conda-forge/{feedstock}-feedstock"
+            )
+            r.raise_for_status()
+
+
+def run(request):
+    action = request["action"]
+    assert action in ("add_feedstock_output",)
```
Comment (jaimergp) [inline conda_forge_admin_requests/feedstock_outputs.py:62]:
```suggestion
    assert action == "add_feedstock_output"
```

File: conda_forge_admin_requests/feedstock_outputs.py [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+    assert "feedstock_to_output_mapping" in request
```
Comment (jaimergp) [inline conda_forge_admin_requests/feedstock_outputs.py:64]: Same here: not empty checks?

**Review by jaimergp — APPROVED** [review]
Just a couple comments, non blocking. Sorry for the brevity, typing from mobile :)

File: conda_forge_admin_requests/feedstock_outputs.py (hunk 5 above)
Comment (beckermr) [reply conda_forge_admin_requests/feedstock_outputs.py:64]:
```suggestion
    assert request.get("feedstock_to_output_mapping")
```

**Pull request merged**